cirq-core/cirq/ops/measure_util.py

```diff
@@ -28,6 +29,56 @@ def _default_measurement_key(qubits: Iterable[raw_types.Qid]) -> str:
     return ','.join(str(q) for q in qubits)
 
 
+def measure_pauli_observable(
+    pauli_observable: pauli_string.PauliString,
+    key: Optional[Union[str, value.MeasurementKey]] = None,
+):
+    """Returns a single PauliMeasurementGate which measures the pauli observable
+
+    Args:
+        pauli_observable: The pauli observable to measure.
+        key: The string key of the measurement. If this is None, it defaults
+            to a comma-separated list of the target qubits' str values.
+
+    Returns:
+        An operation measuring the pauli observable.
+
+    Raises:
+        ValueError: if the observable is not an instance of PauliString.
+    """
+    if not isinstance(pauli_observable, pauli_string.PauliString):
+        raise ValueError(
+            f'Pauli observable {pauli_observable} should be an instance of cirq.PauliString.'
+        )
+    if key is None:
+        key = _default_measurement_key(pauli_observable)
+    return PauliMeasurementGate(pauli_observable.values(), key).on(*pauli_observable.keys())
+
+
+def measure_in_pauli_basis(
```

Comment:
Ahhh that's a good point I hadn't thought about that. I'm still not a huge fan of the old names, but now I'm not a fan of the `measure` + `measure_each` suggestions either lol. How about:
`measure_paulistring` and `measure_paulistring_terms` ? They hint strongly that the input is a paulistring type and that the latter breaks up each term in the paulistring to a seperate basis transformation + measurement operation.

Reply:
drive by comment: I couldn't tell from reading the names what the distinction would be. Is it just one vs many? I would make that distinction clear in the name.

we should also use the name "pauli string" in the names, since a pauli string is a very restricted type of observable. contrast with paulisum

Reply:
How about `measure_paulistring_observable` and `measure_paulistring_terms` ? 

For me, measuring an observable hints towards it being a single measurement, and it helps differentiate with measuring each term individually.

Reply:
why hint? why not say it: `measure_single_paulistsring` `measure_paulistrings`

Reply:
Maybe `measure_single_paulistring` and `measure_paulistring_terms` ? 

`measure_paulistrings` can mean it'll take a list of pauli strings to measure instead of measuring each term of a pauli string.

Reply:
yeah good point

Reply:
I've changed the names to `measure_single_paulistring` and `measure_paulistring_terms`.